tests/fabric_gen_test/configmem_test/test_configmem_rtl.py

```diff
@@ -59,9 +59,8 @@ async def initialize_configmem(dut: ConfigMemDUT) -> None:
     await Timer(10, units="ps")
 
 
-@pytest.mark.skip(reason="Cocotb test - run by simulation, not pytest")
 @cocotb.test
-async def test_configmem_settings(dut: ConfigMemDUT) -> None:
```

Comment:
why did you rename the test, I thought it is best practice to name tests with test_<>?